index.src.html

```diff
@@ -564,15 +606,47 @@ <h4 id="algorithm-store" algorithm>Store a `Credential`</h4>
 
     4.  Return a {{Promise}} which resolves with the result of executing |credential|'s
         [=interface object=]'s {{Credential/[[Store]](credential)}} internal method on |credential|.
-  </ul>
+  </ol>
+
+  <h4 id="algorithm-create" algorithm>Create a `Credential`</h4>
+
+  The <dfn abstract-op>Create a `Credential`</dfn> algorithm accepts a {{CredentialCreationData}}
+  (|data|), and returns a {{Promise}} which resolves with a {{Credential}} if one can be created
+  using the options provided, or with `null` if not.
+
+  <ol class="algorithm">
+    1.  Let |settings| be the <a>current settings object</a>
+
+    2.  Assert: |settings| is a [=secure context=].
+
+    3.  Let |interfaces| be the set of |data|' <a>relevant credential interface objects</a>.
+
+    4.  Return a {{Promise}} rejected with `NotSupportedError` if any of the following statements
+        are true:
+
+        1.  |settings| does not have a [=environment settings object/responsible document=].
```

Comment:
I try to avoid that just so we can talk about item 1.2.2.1.5 if we need to. *shrug* Personal taste?